.npmrc

```diff
@@ -0,0 +1,2 @@
+registry=https://registry.npmjs.org/
```

Comment:
This file is in `.gitignore`. If we add it we should comment that line out. I'm not sure we should add this though, it's not a standard pattern I've seen.